### PR TITLE
One hot encoding, fully-connected và logit.

### PR DESCRIPTION
**Pull request by lkhphuc**
#732 có nhắc tới từ này. Mình đề xuất cũng giữ nguyên không dịch `One-hot`.
Update: Hiện mình đang giữ nguyên 3 từ này.
Context của `fully-connected`: https://github.com/aivivn/d2l-vn/pull/738/files#diff-176d4b05eecbf6877051af34210f4ed2R139

### CODE REVIEW

**Comment by lyphilong** [comment]
em thì nghĩ mình giữ nguyên luôn ạ, không dịch từ này và coi nó như một tên riêng luôn ạ

**Comment by lkhphuc** [comment]
Mình mới thêm từ `logit` vô PR luôn, cảm phiền @duythanhvn và @lyphilong vô review lại giúp nhé.

File: glossary.md [hunk 1]
```diff
@@ -169,6 +169,7 @@ Nếu bạn cho rằng một từ không nên dịch ra tiếng Việt, bạn c
 | linear discriminant analysis (LDA) | phân tích biệt thức tuyến tính |                       |
 | Long Short-term Memory (LSTM)      | bộ nhớ ngắn hạn dài            |                       |
 | logistic regression                | hồi quy logistic               |                       |
+| logit (in softmax)                 | logit                          |                       |
```
Comment (duythanhvn) [inline glossary.md]:
```suggestion
| logit (trong softmax)                 | logit                          |                       |
```

**Comment by lyphilong** [comment]
@lkhphuc  @duythanhvn  em nghĩ về phần hàm logit thì em nghĩ cũng hông nên dịch luôn, nếu có dịch thì em có đề xuất dịch thành "lôgit", tại em cũng thấy nó giống như hàm softmax, sigmoid hay leakyRelu ạ :))

**Comment by duythanhvn** [comment]
@lyphilong được rồi nà, để nguyên thôi, dịch kiểu A-lét-xăng-đơ thì.... toang lắm bố cháu ạ @@

**Comment by lkhphuc** [comment]
Sẵn tiện ae đang ở đây `fully-connected` trước giờ mình có dịch chưa nhỉ?

**Comment by duythanhvn** [comment]
Ý của @lkhphuc là nó đã xuất hiện trong cuốn sách này chưa hay là anh em đã từng dùng từ này chưa?

**Comment by lkhphuc** [comment]
Ý là trong sách này đã dịch từ này chưa hay mình giữ nguyên? Đang không biết dịch sao?

**Comment by lkhphuc** [comment]
@duythanhvn @lyphilong hứa không nhét thêm gì vô PR này nữa. Mọi người thảo luận cái nhé :)).

**Comment by duythanhvn** [comment]
Trong sách chưa dịch, từ này xuất hiện ở đâu nhỉ, thí chủ có thể cho xin "cái linh" để xem đẩy đủ hông.

**Comment by lyphilong** [comment]
Em thì vote không dịch từ đó, mà nếu dịch thì từ đó đồng nghĩa với dense layer nên em có đề xuất dịch là "tầng dày đặc"

**Comment by lkhphuc** [comment]
Fully-connected ở đây nhé: https://github.com/aivivn/d2l-vn/pull/738/files#diff-176d4b05eecbf6877051af34210f4ed2R139

**Comment by duythanhvn** [comment]
Ờm.... từ này đúng là chưa biết dịch như thế nào... dịch mà sát sàn sạt bảo là `kết nối đầy đủ` thì anh em thấy có hợp lý được không? 

Nhớ ở dự án trước anh Tiệp có share một cái Google Sheets bảng thuật ngữ của team Deep Learning, không biết @lkhphuc có bookmark lại không, mình tìm lại mà không còn thấy nữa, nên hỏi xem có từ này không.

**Comment by lkhphuc** [comment]
https://docs.google.com/spreadsheets/d/1Hcd4AqR7Xzd08Ws0gwhZ-LX6XNzJKgCDy0VcFc3D66c/edit#gid=0 
Nó đây @duythanhvn , chưa thấy từ này trong đó.

File: glossary.md [hunk 2]
```diff
@@ -116,6 +116,7 @@ Nếu bạn cho rằng một từ không nên dịch ra tiếng Việt, bạn c
 | fit                 | khớp           |               |
 | first principle     | định đề cơ bản |               |
 | functional anaylsis | giải tích hàm  |               |
+| fully-connected     | fully-connected  |               |
```
Comment (lkhphuc) [inline glossary.md]:
```suggestion
| fully-connected     | kết nối đủ        |               |
```

Comment (lkhphuc) [inline glossary.md]:
```suggestion
| fully-connected     | fully-connected  |               |
```

Comment (duythanhvn) [reply glossary.md]: hay là... hoàn toàn?

Comment (lkhphuc) [reply glossary.md]: Nếu dịch thì t vẫn thích `đủ` hoặc `đầy đủ` hơn, chỉ phân vân có nên dịch không.

Comment (duythanhvn) [reply glossary.md]: Mình thấy nên dịch, thêm ý kiến nhỉ. 
cc @minhduc0711 @rootonchair @tiepvupsu

Comment (duythanhvn) [reply glossary.md]: Nếu mà dịch thì mình vote từ `đầy đủ` hơn...

Comment (rootonchair) [reply glossary.md]: Em vote "kết nối đầy đủ" nhé

File: glossary.md [hunk 3]
```diff
@@ -206,6 +208,7 @@ Nếu bạn cho rằng một từ không nên dịch ra tiếng Việt, bạn c
 | orthogonal         | trực giao          |                       |
 | orthonormal        | trực chuẩn         |                       |
 | overfit            | quá khớp           | http://bit.ly/2BvfPYA |
+| one-hot encoding   | mã hoá one-hot     |                       |
```
Comment (tanduong) [inline glossary.md]: Chỗ này cần team cân nhắc lại. Khi dịch ra tiếng việt thì encode và encrypt đều dịch ra là mã hóa. Nhưng đây là 2 hành động có động cơ và kỹ thuật rất khác nhau.

Encode là chuyển dữ liệu từ định dạng này qua định dạng khác mà vẫn bảo toàn thông tin.
Encrypt là mã hóa dữ liệu để người không được phép không đọc được (không có chìa khóa).

Chữ encode có thể dịch là `biểu diễn` hoặc `số hóa`.

Comment (tanduong) [reply glossary.md]: Và thật ra, trong project này không nên có chữ "mã hóa" nào cả.

File: glossary.md (hunk 2 above)
Comment (minhduc0711) [reply glossary.md]: mình cũng thấy `kết nối đầy đủ` ổn

File: glossary.md (hunk 3 above)
Comment (minhduc0711) [reply glossary.md]: +1 cho `biểu diễn`

Comment (lkhphuc) [reply glossary.md]:
```suggestion
| one-hot encoding   | biễu diễn one-hot     |                       |
```

File: glossary.md (hunk 2 above)
Comment (lkhphuc) [reply glossary.md]: Mọi người đi qua cho cái emoji vote nhé:
- Kết nối đầy đủ 👍 
- Kết nối hoàn toàn ❤️ 
- Không dịch 🎉

File: glossary.md [hunk 4]
```diff
@@ -206,6 +208,7 @@ Nếu bạn cho rằng một từ không nên dịch ra tiếng Việt, bạn c
 | orthogonal         | trực giao          |                       |
 | orthonormal        | trực chuẩn         |                       |
 | overfit            | quá khớp           | http://bit.ly/2BvfPYA |
+| one-hot encoding   | biễu diễn one-hot     |                       |
```
Comment (tiepvupsu) [inline glossary.md]: Chỗ này sai thứ tự bảng chữ cái từ trước rồi. `one` phải trước `orthogonal` chứ nhỉ.
